### PR TITLE
docs: add quick guide to the charts' README

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## WHAT

Improves the `README.md` files (or rather: the `*.gotmpl`) of our helm charts explaining some details about SSI.

## WHY

better documentation

## FURTHER NOTES

- removed the `controlplane.ssi.endpoint.audience` setting, as this always defaults to the callback URL

Closes # <-- _insert Issue number if one exists_

### CODE REVIEW

**Comment by paullatzelsperger** [comment]
\cc @jimmarino

**Review by wolf4ood — APPROVED** [review]

File: charts/tractusx-connector-azure-vault/README.md [hunk 1]
```diff
@@ -9,31 +9,38 @@ This chart is intended for use with an _existing_ PostgreSQL database and an _ex
...
+## Setting up SSI
 
-- `daps-cert`: contains the x509 certificate of the connector.
-- `daps-key`: the private key of the x509 certificate
-- `aes-keys`: a 128bit, 256bit or 512bit string used to encrypt data. Must be stored in base64 format.
+### Preconditions
 
-These must be obtained from a DAPS instance, the process of which is out of the scope of this document. Alternatively,
-self-signed certificates can be used for testing:
+- the Managed Identity Walled (MIW) must be running and reachable via network
```
Comment (jimmarino) [inline charts/tractusx-connector-azure-vault/README.md:16]: channeling my inner Grammarly: "via the network"

File: charts/tractusx-connector-azure-vault/README.md [hunk 2]
```diff
@@ -9,31 +9,38 @@ This chart is intended for use with an _existing_ PostgreSQL database and an _ex
...
+- the necessary set of VerifiableCredentials for this participant must be pushed to MIW. This is typically done by the
+  Portal during participant onboarding
+- KeyCloak must be running and reachable via network
+- an account with KeyCloak must be created for this BPN and the connector must be able to obtain access tokens
+- the client ID and client secret corresponding to that account must be known
```
Comment (jimmarino) [inline charts/tractusx-connector-azure-vault/README.md:21]: Keycloak account

**Review by jimmarino — COMMENTED** [review]
LGTM,  just a few minor comments

**Pull request merged**